suite-common/formatters/src/kinds/prepareCryptoAmountFormatter.ts

```diff
@@ -0,0 +1,58 @@
+import BigNumber from 'bignumber.js';
+
+import { networksCompatibility as NETWORKS, NetworkSymbol } from '@suite-common/wallet-config';
+import {
+    localizeNumber,
+    networkAmountToSatoshi,
+    formatCoinBalance,
+} from '@suite-common/wallet-utils';
+import { PROTO } from '@trezor/connect';
+
+import { makeFormatter } from '../makeFormatter';
+import { FormatterConfig } from '../types';
+
+export type CryptoAmountFormatterInputValue = string | number | BigNumber;
+
+export type CryptoAmountFormatterDataContext = {
+    isBalance?: boolean;
+    symbol?: string;
```

Comment:
I think we should have better type for symbol

Reply:
https://github.com/trezor/trezor-suite/pull/6235/commits/6d1c1bd5008a23e3a6bfc1caaa9da9d12f7be900